src/components/ui/NavBar.tsx

```diff
@@ -0,0 +1,80 @@
+import { type FC, type PropsWithChildren } from "react";
+import { type IconType } from "react-icons/lib";
+import { TbScubaMask, TbHistory, TbAward } from "react-icons/tb";
+
+type NavItemProps = {
+  title: string;
+  Icon: IconType;
+  pageNo: number;
+  pageHandler: (pageNo: number) => void;
+};
+
+type NavBarProps = {
+  page: number;
+  pageHandler: (pageNo: number) => void;
+};
+
+const NavItem: FC<PropsWithChildren<NavItemProps>> = ({
+  children,
+  title,
+  Icon,
+  pageNo,
+  pageHandler,
+}) => {
+  const onChangePage = () => {
+    pageHandler(pageNo);
+  };
+  return (
+    <div
+      className="relative flex flex-1 flex-col justify-center"
+      onClick={onChangePage}
+    >
+      {children}
+      <Icon size={36} className="mx-auto text-secondary" />
+      <p className="text-center text-sm font-semibold text-secondary">
```

Comment:
I think you can also either change colour or bold the selected tab - just in case the bump is not obvious to some users

<img width="385" alt="image" src="https://user-images.githubusercontent.com/63008910/227420184-eeaeb483-badb-4242-9288-20539c2ef730.png">